config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
-You have done well so far, but according to the project requirements, you are to setup your routes for each controller action. Please ensure you maintain RESTful URL route. i.e do not expose unnnecessary endpoints.

Reply:
noted thanks